robottelo/cli/contentview.py

```diff
@@ -117,7 +117,7 @@ def publish(cls, options, timeout=None):
         cls.command_sub = 'publish'
         # Publishing can take a while so try to wait a bit longer
         if timeout is None:
-            timeout = 120
+            timeout = 600
```

Comment:
Why 600? For api we have 300 secs for everything and 900 for few commands, affected by some BZ. I think we should use the same approach here not to introduce differences in api/cli behavior. And btw, `publish` is not the only command which takes more than 2 minutes, please update others accordingly.